components/KebabPopUpController.tsx

```diff
@@ -66,6 +67,10 @@ export function useKebabPopUp(props) {
   let isWalletBindingError = useSelector(service, selectShowWalletBindingError);
   let otpError = useSelector(service, selectOtpError);
   let walletBindingError = useSelector(service, selectWalletBindingError);
+  let bindingAuthFailedError = useSelector(
```

Comment:
It can const as it's not changed later

Reply:
@swatigoel we are replacing it with esignet machine function if we are using openId4VCI. So we have to keep it as let

Reply:
@swatigoel i think we don't need this if condition for openI4VCI becuase in props we are already getting exisitngMosipVcItem machine or EsignetMosipVcItemMachine as props and when we call the functions it will call the corresponding machine functions only so we don't need this if condition. if there is any function which is present in one machine but not in other we can handle those functions in if condition. I will check with @vijay151096 regarding this as he added the code

Reply:
I have extracted the common functions of EsignetMosipVcItemMachine & exisitngMosipVcItemMachine into common file and removed if condition from kebabPopupController

Reply:
Validate this change once with @vijay151096

Reply:
validated.